### PR TITLE
eligibility-util: submission

### PR DESCRIPTION
**Pull request by i0ntempest**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: sysutils/eligibility-util/Portfile [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+description         Open source macOS system eligibility tweaking utility
+long_description    ${description}. Can be used for enabling features unavailable in certain Mac SKUs, \
+                    e.g. region locked features like Apple Intelligence.
+
+checksums           rmd160  fcd6158935b6b0e84a06e57800fe6a6e9b4fbca8 \
+                    sha256  56bd6be6de38654a78bccd370d2f2cfa9fb2b3dd950f794c2f1b326101ba9b34 \
+                    size    380169
+
+if {${os.major} < 24} {
```
Comment (reneeotten) [inline sysutils/eligibility-util/Portfile:23]: as usual, this should also check for os.platform

Comment (i0ntempest) [reply sysutils/eligibility-util/Portfile:23]: Doesn’t the platforms line at the top already restricts installation to macOS only?

File: sysutils/eligibility-util/Portfile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+}
+
+xcode.project       ${github.project}.xcodeproj
+xcode.target        eligibility_util
+xcode.scheme        ${xcode.target}
+xcode.configuration Release
+
+pre-build {
+    #system -W ${worksrcpath} "swift build --disable-sandbox"
```
Comment (markemer) [inline sysutils/eligibility-util/Portfile:37]: Another thing to add to my TODO list - a swift pm portgroup

File: sysutils/eligibility-util/Portfile (hunk 1 above)
Comment (reneeotten) [reply sysutils/eligibility-util/Portfile]: You're probably right, I missed that one (and assumed it was the default `darwin`)

**Comment by reneeotten** [comment]
please feel free to reopen once this is in state that it can be committed.

**Comment by i0ntempest** [comment]
This is another PR that needs a new fixed Xcode/SwiftPM pg.

**Comment by reneeotten** [comment]
then please close that as well until it’s ready to commit. The PR queue is not intended to park unfinished things.

**Pull request closed without merging**